docs/griptape-framework/structures/tasks.md

```diff
@@ -596,3 +596,38 @@ task = OutpaintingImageGenerationTask(
 
 task.run()
 ```
+
+## Image Query Task
+
+The [Image Query Task](../../reference/griptape/tasks/image_query_task.md) executes a natural language query on one or more input images. This Task uses an [Image Query Engine](../engines/image-query-engines.md) configured with an [Image Query Driver](../drivers/image-query-drivers.md) to perform the query. The functionality provided by this Task depend on the capabilities of the model provided by the Driver.
+
+This Task accepts two inputs: a query (represented by either a string or a [Text Artifact](../data/artifacts.md#textartifact)) and a list of [Image Artifacts](../data/artifacts.md#imageartifact) or a Callable returning these two values.
+
+```python
+from griptape.engines import ImageQueryEngine
+from griptape.drivers import OpenAiVisionImageQueryDriver
+from griptape.tasks import ImageQueryTask
+
+
+# Create a driver configured to use OpenAI's GPT-4 Vision model.
+driver = OpenAiVisionImageQueryDriver(
+    max_tokens=100,
```

Comment:
I remember that this `max_tokens` was pretty important for this functionality. Is that specific to the Vision Driver or all Drivers? Is that something we want to call out here (or in the Driver docs?)

Reply:
That is a specific quirk of the Vision driver and it's mentioned on its page.